mobile/evidence/ios/Features/Tests/LoginTests/LoginEmailViewModelTests.swift

```diff
@@ -11,101 +11,102 @@ import XCTest
 
 final class LoginEmailViewModelTests: XCTestCase {
 
+    func testConfirmationPopupAppears() {
+        let viewModel = LoginEmailViewModel()
+        viewModel.confirmationPopupAppears()
+        let expectation = XCTestExpectation(description: "Confirmation popup should appear")
+        
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
+            XCTAssertEqual(viewModel.offSetY, 1000, "OffsetY should be updated to 1000")
```

Comment:
Let's talk about this delay in tests! The swift-dependencies package ships with a [clock](https://github.com/pointfreeco/swift-dependencies?tab=readme-ov-file#quick-start) dependency that can be used to allow waiting some time to pass during user experience, but allowing it to happen instantly in unit tests.

Reply:
yes chef!